src/components/Login/Login.tsx

```diff
@@ -51,19 +52,28 @@ const Login = () => {
   const [key, setKey] = useState(null);
   const [seed, setSeed] = useState(null);
   const [showSpinner, setShowSpinner] = useState(false);
+  const [address, setAddress] = useState(null);
```

Comment:
Not a big fan of this entire flow, it's a bit convoluted and there are a lot of (probably) unnecessary re-renders.

Like why is the second `useEffect` caring about `key` and `seed`, but then if those things are true it is just returning and not performing any action? Not intuitive at all even if it is actually correct.

Also regardless should be unregistering the listener in the first `useEffect`'s cleanup function – but probably better would just be to find a way to include this code in the new `BitgoOrchestrator` component, and have all the associated state for this flow be in the redux store.